djangocms_pageadmin/static/djangocms_pageadmin/js/actions.js

```diff
@@ -4,6 +4,95 @@
     }
 
     $(function() {
+
+        var createBurgerMenu = function(row) {
+
+            // create anchor icon
+            var anchor = document.createElement('A');
+            var cssclass = document.createAttribute('class');
+            cssclass.value = 'btn cms-page-admin-action-btn closed';
+            anchor.setAttributeNode(cssclass);
+            var title = document.createAttribute('title');
+            title.value = 'Actions';
+            anchor.setAttributeNode(title);
+            var icon = document.createElement('IMG');
+            var src = document.createAttribute('src');
+            src.value = '/static/djangocms_pageadmin/svg/menu.svg'
```

Comment:
This won't work if your static files are served from external domain

Reply:
Any suggestions? I tried this but it doesn't work because `actions.js` is not being included by our template. Not sure what includes it at this stage.

Reply:
Looks like it's being included via `admin.py`: 
```
    class Media:
        js = ("djangocms_pageadmin/js/actions.js",)
        css = {"all": ("djangocms_pageadmin/css/actions.css",)}
```
Not sure how to pass the static path as a variable to that.

Reply:
ok figured it out... just moved the script inclusion into the template.